mod/node-api/backend/blob.go

```
@@ -0,0 +1,94 @@
+	blobSidecarsResponse := make([]*beacontypes.BlobSidecarData[BeaconBlockHeaderT], 0, responseSize)
+
+	// Process each sidecar
+	for i := 0; i < blobSidecars.Len(); i++ {
+		blobSidecar := blobSidecars.Get(i)
+
+		// Skip if indices specified and this index not requested
+		if len(indices) > 0 && !indexMap[blobSidecar.GetIndex()] {
+			continue
+		}
+
+		blobHex, err := blobSidecar.GetBlob().MarshalText()
```

golangci-lint

shadow: declaration of "err" shadows declaration at line 32

(govet)